docs/admin/setup.rst

```diff
@@ -0,0 +1,70 @@
+.. _ref_admin_setup:
+
+=============
+Initial Setup
+=============
+
+Once EdgeDB has been installed using one of the methods described in the
+:ref:`ref_admin_install` section, a couple of initial setup steps are
+necessary in order to start using it.
+
+
+Creating an EdgeDB Instance
+===========================
+
+.. note::
+
+    If you have installed EdgeDB using a pre-built package, a default
+    system-wide instance would have been created
+    (usually in ``/var/lib/edgedb-N/``, where ``N`` is the major version
+    of EdgeDB).  In this case you can skip this section and refer to
+    the installation section appropriate for your OS.
+
+To use EdgeDB you must first create an *instance*.  An EdgeDB instance
+is a collection of *databases* that is managed by a particular running
+EdgeDB server.  The data managed by the instance is usually stored in
+a single directory, which is called the *data directory*.
+
+The EdgeDB server creates and populates a data directory automatically
+on the first run.  The location of the data directory can be specified
+using the ``-D`` option:
+
+.. code-block:: bash
+
+    $ edgedb-server -D /data/directory/path
```

Comment:
Is there a difference between `edgedb-server` and `edb server`?

Reply:
`edb` is the develoment-mode command, we don't install it in production builds.